### ISSUE TITLE
roguelike about being the only woman in a tech conference dungeon

### ISSUE THREAD

**Issue by pdurbin** [GinBaby/hands-on-haskell-meetups#2]
At around 47:40 at https://www.functionalgeekery.com/episode-107-julie-moronuki/ I heard, "I'm writing a roguelike right now where the dungeon is a tech conference and your character is the only woman at the tech conference." Is this the right repo? Thanks!

**Comment by argumatronic** [comment]
Yes, when it's finished I'm going to document its building for people to use in learning/teaching Haskell, and all that will be in this repo. Development has stalled a bit lately, but now that I talked about it publicly, I guess I better get it going again.

**Comment by pdurbin** [comment]
Cool, no rush. I just thought some of the groups I'm friendly with such as [Harvard Women In Tech](https://wit.abcd.harvard.edu) or [Girl Hackathon](http://girlhackathon.com) would get a kick out of it. Fun episode. I'm going to have to try out https://code.world with my kids and other stuff you mentioned.
